### PR TITLE
Print std_output if it's not empty

### PR DESCRIPTION
**Pull request by jhshi**

### CODE REVIEW

File: leetcode/views/result.py [hunk 1]
```diff
@@ -82,6 +82,10 @@ def make_failed_view(self):
                 blank, your_answer_header, your_answer,
                 blank, expected_answer_header, expected_answer
         ]
+        if len(self.result.get('std_output', '')) > 0:
+            stdout_header = urwid.Text('Stdout:')
+            stdout = urwid.Text(self.result['std_output'])
+            list_items.extend([blank, stdout_header, stdout])
```
Comment (chishui) [inline leetcode/views/result.py]: Is the std_output supposed to display at other result?

Comment (jhshi) [reply leetcode/views/result.py]: Typically std_output is interesting while debugging failed tests. We can add it to other views as well but I don't think that's necessary.

**Comment by jhshi** [comment]
Actually on a second thought, maybe we should print stdout whenever it's not empty in any view. PR updated accordingly.

File: leetcode/views/result.py (hunk 1 above)
Comment (chishui) [reply leetcode/views/result.py]: I tried to output some content to std output in my code, the "std_output" can be very large sometime that I don't think it useful to display them all. Maybe we should restrict the std_output size?

Comment (jhshi) [reply leetcode/views/result.py]: That's a good idea. PR updated.

**Pull request merged**